policyengine_canada/variables/gov/provinces/nu/tax/income/credits/age_amount/nu_age_amount_credit.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_canada.model_api import *
+
+
+class nu_age_amount_credit(Variable):
+    value_type = float
+    entity = Person
+    label = "Nunvaut age amount credit"
+    unit = CAD
+    definition_period = YEAR
+    defined_for = ProvinceCode.NU
```

Comment:
And remove from formula
```suggestion
    defined_for = "nu_age_amount_credit_eligible_person"
```